src/main/kotlin/no/nav/familie/ef/sak/simulering/SimuleringService.kt

```diff
@@ -83,6 +83,16 @@ class SimuleringService(
         )
     }
 
+    fun simuleringOppsummeringerErUlike(saksbehandling: Saksbehandling): Boolean {
```

Comment:
```suggestion
    fun erSimuleringsoppsummeringEndret(saksbehandling: Saksbehandling): Boolean {
```
Vi pleier å starte med "er" først

Reply:
Ville også laget en test på denne metoden.